kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html

```diff
@@ -1,5 +1,4 @@
-<div class="panel panel__left panel--side"
-     ng:class="{ 'panel__left--hidden': !ctrl.isVisible }">
+<div>
```

Comment:
It's another refactoring with the class names because of the image page.
I was hoping to do that separately (i.e. move the image page over to use the same panelled view).